File: VB/DateAndTimeColumns_Unbound/Item.vb
Imports DevExpress.Mvvm

Namespace DateAndTimeColumns_Unbound

    Public Class Item
        Inherits BindableBase

        Private dateTimeField As Date

        Public Property DateTime As Date
            Get
                Return dateTimeField
            End Get

            Set(ByVal value As Date)
                dateTimeField = value
                RaisePropertiesChanged()
            End Set
        End Property
    End Class
End Namespace

File: CS/DateAndTimeColumns_Unbound/Item.cs
using DevExpress.Mvvm;
using System;

namespace DateAndTimeColumns_Unbound
{
    public class Item : BindableBase
    {
        private DateTime dateTime;
        public DateTime DateTime {
            get => dateTime;
            set {
                dateTime = value;
                RaisePropertiesChanged();
            }
        }
    }
}
